app.py

```diff
@@ -1,22 +1,26 @@
 from flask import Flask, render_template, jsonify, request, url_for
 import json
 import pandas as pd
-import retrieve_definition
 from retrieve_definition import retrieve_definition
 
 
+# Create Flask app
 app = Flask(__name__)
 
 
+# Creating the main route for the api page
 @app.route('/', methods=['GET', 'POST'])
 def home():
     """ Home page of site """
     message = "You are now home"
     return render_template('base.html', message=message)
 
 
+# Creating a serach route to access retrieve_defenition function
 @app.route('/search', methods=['GET', 'POST'])
 def wiki_search():
+    """Accessing wikipedia's api with 
+    retrieve_defenition function"""
```

Comment:
spelling error here, should be retrieve_definition

Reply:
Thank you, I resolved the error!!